docs/scan.md

```diff
@@ -0,0 +1,86 @@
+# Scanning
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+-   [How It Works](#how-it-works)
+    -   [Pre-commit Hook](#pre-commit-hook)
+    -   [Baseline File](#baseline-file)
+        -   [Notable Fields](#notable-fields)
+-   [What Gets Scanned?](#what-gets-scanned)
+    -   [Secret Verification](#secret-verification)
+-   [How It’s Used](#how-its-used)
+-   [Excluding Files](#excluding-files)
+-   [Plugins](#plugins)
+-   [Adjusting the Scan Sensitivity](#adjusting-the-scan-sensitivity)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## How It Works
+
+`detect-secrets scan` scans the entire codebase and outputs a snapshot of currently identified secrets.
+
+This snapshot should be stored in a baseline file and updated on an as-needed basis. The `detect-secrets-hook` - i.e. pre-commit hook - will notify you when your baseline file needs to be updated.
+
+### Pre-commit Hook
+
+The pre-commit hook uses `detect-secrets`'s scanning functionality to scan your code before it gets committed. It's recommended to set up this hook ([docs](./docs/../developer-tool-faq.md#how-do-i-set-up-the-pre-commit-hook)) to prevent leaks before they reach GitHub.
```

Comment:
```suggestion
The pre-commit hook uses `detect-secrets`'s scanning functionality to scan your code before it gets committed. It's recommended to set up this hook ([docs](./developer-tool-faq.md#how-do-i-set-up-the-pre-commit-hook)) to prevent leaks before they reach GitHub.
```